src/lib/components/CommunitySelectionModal/CommunityListItem.js

```diff
@@ -12,70 +12,57 @@ import PropTypes from 'prop-types';
 import { i18next } from '@translations/i18next';
 import { CommunityContext } from './CommunityContext';
 
-export const CommunityListItem = ({ result, standAlone }) => {
+export const CommunityListItem = ({ result }) => {
   const { setLocalCommunity, getChosenCommunity } =
     useContext(CommunityContext);
 
   const { metadata } = result;
   const linkToCommunityPage = result.links.self_html;
   const linkToLogo = result.links.logo;
-
   const itemSelected = getChosenCommunity()?.uuid === result.uuid;
 
   return (
-    <Item key={result.id} className="pr-20">
+    <Item key={result.id} className={itemSelected ? 'selected' : ''}>
       <Item.Image
-        as={Image}
         size="tiny"
-        src={linkToLogo}
-        fallbackSrc="/static/images/square-placeholder.png"
+        src={linkToLogo || "/static/images/square-placeholder.png"}
```

Comment:
Why was this needed? I think the fallbackSrc is actually there in case the linkToLog is giving an error...Doesn't work as expected?

Reply:
No, the fallback image didn't show up? But this solved the issue.

Reply:
I think the fallback image is broken in the original component and this is only the workaround. It should be fixed in the source, the Image component of invenio-react-forms was implemented to automatically solve the issue of default image placeholder

Reply:
Will change it back!

Reply:
@jennur Is it working after that change or the image is broken?

Reply:
I checked the PR locally and the images are not shown properly. The issue should not be fixed here so I created an issue https://github.com/inveniosoftware/react-invenio-deposit/issues/482 . @jennur let me know if you want to investigate on this :) I will merge this PR as it is blocking https://github.com/inveniosoftware/invenio-app-rdm/issues/1483 .